index.html

```diff
@@ -0,0 +1,122 @@
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+ <link href="https://fontawesome.com/icons/chevron-down?f=classic&s=regular">
+
+</head>
+<body>
+  <main>
+
+          <header >
+            <nav class="navbar">
+
+             <div class="hambuger">
+              <span class="bar"></span>
+              <span class="bar"></span>
+              <span class="bar"></span>
+             </div>
+
+             <ul class="nav-menu">
+              <li class="nav-items"><a href="index.html">Home</a></li>
+              <li class="nav-items"><a href="about.html" >About Us</a></li>
+              <li class="nav-items"><a href="#" >Courses</a></li>
+              <li class="nav-items"><a href="#" >Contact</a></li>
+            </ul>
+            </nav>
+
+            <div class="header-1">
+              <nav class="header-1d">
+                <ul>
+                  <li><a href=""></a><i class="fa fa-facebook"></i></li>
+                  <li><a href=""></a><i class="fa fa-twitter"></i></li>
+                  <li><a href=""></a>English</li>
+                  <li><a href=""></a>My Page Layout</li>
+                  <li><a href=""></a>Logout</li>
+                </ul>
+              </nav>
+            </div>
+            <div class="header-2">
+              <nav class="desk-nav">
+                <div class="logo1">  <a href="#home"> <img src="./pics/partners/cck-logo1 1.png"> </a> </div>
+                <ul class="nav-d">
+                  <li class="item"><a href="about.html">About</a></li>
+                  <li class="item"><a href="index.html">Program</a></li>
+                  <li class="item"><a href="#">Join</a></li>
+                  <li class="item"><a href="#">Sponsor</a></li>
+                  <li class="item"><a href="#">Updates</a></li>
+                </ul>
+                <div class="logo2"> a Campaign</div>
+              </nav>
+            </div>
+          </header>
+
+          <section  class="home"> <a name="home"></a>
+              <div class="home-intro">
+                <p>"Hello! Kid Developers"</p>
+                <h1> INFORMATION TECHNOLOGY  <br> FOR  KIDS  <br>  <p>2023</p></h1>
+
+              </div >
+              
+              <div class="intro-p">
+                <p> <u>Our vision</u>  : <strong> a source of empowerment for  digital education for children</strong>  accross Africa.
+                  We are open for the vulgarisation of digital education and arts creation for all our kids. As from:<strong> July 2023</strong>
+                </p>
+              </div>
+        
+            <div class="intro-dat">
+              <h2> 2023.03.07 (MON) ~ 07 (FRI)  </h2>
+              <p>&#64; The Afican kid's-Tech Center, Art Centre and more.</p>
+            </div>
+       </section>
+       <section>
+            <div id="home-container">
```

Comment:
- [ ] Kindly adhere to the project requirement as I can see you are still not following the instructions given by the previous code reviewer which is the button at the bottom, which is a crucial element according to the design, is missing. It is important to include the button as specified in the design. Keep up the good work and attention to detail!

### Requirement
![image](https://user-images.githubusercontent.com/66476329/236188683-4224cc77-237d-413d-9942-91297c87687b.png)

### Design Comparison
<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![image](https://user-images.githubusercontent.com/55840999/251867691-da6fa216-e6d3-45e4-9c54-e34a80440d23.png)

</td>
<td>

![image](https://github.com/hydaline-code/Capstone_project_1/assets/66476329/86a8f4fc-99d1-4990-b6ed-125faa0a3fef)

</td>
</tr>
</table>